### PR TITLE
refactor: email service

### PR DESCRIPTION
**Pull request by niall-shaw**
https://github.com/didx-xyz/ssi-trust-registry/pull/198#discussion_r1447619185
Addressing the comment made above

### CODE REVIEW

File: packages/backend/src/email/client-stub.ts [hunk 1]
```diff
@@ -42,6 +44,28 @@ export function createEmailClientStub(): EmailClientStub {
...
+      const { submitUiUrl } = getSubmitUrls(invitation)
+      await this.sendMailFromTemplate(
+        invitation.emailAddress,
+        'Sorry. Your submission has been rejected.',
+        './src/email/templates/rejected.html',
+        {
+          submitUiUrl,
+        },
+      )
+    },
```
Comment (jakubkoci) [inline packages/backend/src/email/client-stub.ts:68]: I already discuss this with @andrii-uhryn-absa He was supposed to communicate it with you, but you were on vacation, and the next day, yesterday, he was on vacation :)

We basically duplicate business logic here, that doesn't make any sense. We have to figure out a better way.

Maybe, creating some kind of `EmailService` for these functions, or keep them inside our higher-level modules like submission service and/or invitation service.

**Review by jakubkoci — CHANGES_REQUESTED** [review]

File: packages/backend/src/email/client-stub.ts (hunk 1 above)
Comment (niall-shaw) [reply packages/backend/src/email/client-stub.ts:68]: Sure, I think it makes sense to be in the relevant service, i.e. submission service in this case.
But then that service will have a dependency on email client?

Comment (niall-shaw) [reply packages/backend/src/email/client-stub.ts]: Moved to `emailService`

**Review by jakubkoci — APPROVED** [review]
Nice 👍

**Pull request merged**